### ISSUE TITLE
pretrained weight file

### ISSUE THREAD

**Issue by liujianzhao6328057** [fourson/DeblurGAN-pytorch#1]
I'm happy when i find the pytorch implementation of deblur gan .Could you please upload a copy of pretrained weight file since i want to to test the model quickly.Thank you .

**Comment by fourson** [comment]
I'm glad to hear that you have visited my repo. However the model is still training on my server and I cannot upload a pretrained weight immediately. It may take 6+ days to train this model according to the original paper. So if it is not so urgent, you can wait for my result in about 10 days.

**Comment by liujianzhao6328057** [comment]
hi ，I'm back .Have you got the satisfactory results ?

**Comment by fourson** [comment]
I've got one version of pretrained result, but the test PSNR on the whole GoPro dataset only reached 22. So I changed the model a little bit and retrained it. A bug(the vgg_conv_3_3 in perceptual_loss has been set to loacl variable and it doubled the training time) has been fixed. In addition, I have trained the model using vanilla GAN loss simultaneously, and this result will be uploaded tommorrow(and the result using WGAN_GP loss will be uploaded in about 3-4days).

**Comment by fourson** [comment]
Pretrained weights of GAN_LOSS has been uploaded. And weights of WGAN_GP loss is comming soon.

**Comment by liujianzhao6328057** [comment]
Thank you

**Comment by fourson** [comment]
The weights of WGAN_GP loss have been uploaded.
